### PR TITLE
Adding styles to the page components

### PR DESCRIPTION
**Pull request by o-rashad**
#### What's new ? :new: 
 - Flexbox used to center and align body components
 - Instead of using `main` element as a container i replaced it with `table` element

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,23 +1,32 @@
...
+    </table>
+
+    <hr style="width: 150px;">
+
+    <h1> ➕ Add new book ?</h1>
     <form id="form" action="">
         <input type="text" name="title" id="titleInput" placeholder="📖 Enter book title" required>
         <input type="text" name="author" id="authorInput" placeholder="✍️ Enter author name" required>
         <button type="submit" id="addBookBtn">➕ Add book</button>
     </form>
```
Comment (henatan99) [inline src/index.html]: - [x] I appreciate the creativity displayed in the styling and the overall work is really great. I recommend you wrap these elements with `<main>` tag for better semantics and HTML file structure. The top-level components inside a body are by convention `header`, `main` and `footer`. Kindly make changes here. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @od-c0d3r , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (o-rashad) [reply src/index.html]: Thanks @henatan99, am on it :+1:

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Rashad, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**